src/Uno.UI.RuntimeTests/Uno.UI.RuntimeTests.csproj

```diff
@@ -34,6 +34,7 @@
 	<Import Project="UnitTestsImport.props" />
 
 	<ItemGroup Condition="'$(TargetFramework)'=='netstandard2.0'">
+		<EmbeddedResource Include="Assets\Border_Snapshot.bgra8" />
```

Comment:
You may not want to make the `Border_Snapshot.bgra8` conditional to netstandard 2.0.